docs/mozillavpn.json

```diff
@@ -14,16 +14,26 @@
     "WINDOWS": "2.7.1"
   },
   "releases": {
-    "2.5.1": {
+    "2.0.0": {
       "build": null,
-      "date": "2021-10-20",
-      "platforms": ["ANDROID", "IOS"]
+      "date": "2019-10-04",
+      "platforms": ["ANDROID", "IOS", "LINUX"]
```

Comment:
More platforms may be available for this release but this is all that is needed to display the correct minimum version. (Same with 2.1.0)